.github/workflows/template_release_drafter.yml

```diff
@@ -20,16 +20,39 @@ on:
 jobs:
   update_release_draft:
 
-    name: update release draft
+    name: Update Release Draft
     runs-on: ubuntu-22.04
 
     steps:
+      - name: checkout code
+        uses: actions/checkout@v3
+        with:
+          fetch-depth: 0
+      - name: set version to github env
+        id: set_version
+        run: |
+          # fetch last tag from git
+          OLD_VERSION=`git describe --tags $(git rev-list --tags --max-count=1)`
```

Comment:
In the studio repository we use the command `git tag | sort -V | tail -1` https://github.com/Staffbase/experience-studio/blob/99001ccabb17aac0f1c36fc0779c471d9f889542/.github/scripts/version.js#L4

Reply:
The disadvantage of this command is that you sort it alphabetically. 🤔 So if you change the version structure you could run into issues. With the command above you won't have this issue. 🙌